mbed_app_template.json

```diff
@@ -77,6 +77,17 @@
             "stm32wl-lora-driver.debug_tx": "PA_10",
             "stm32wl-lora-driver.debug_rx": "PA_9",
             "stm32wl-lora-driver.debug_invert": 1
+        },
+        "RAK3172SIP": {
```

Comment:
Not related to this PR ?

Reply:
yeah sorry forgot this one last time so just add this time (but it's template no no real issue)